cli/utils.py

```diff
@@ -88,3 +89,21 @@ def form_path_name(csv_path_format, output_dir):
         timestamp=datetime.now().strftime("%Y%m%dT%H%M%S"),
     )
     return csv_path
+
+
+@main.command()
+@click.argument("csv_path_left", type=str, required=True)
+@click.argument("csv_path_right", type=str, required=True)
+def csv_diff(csv_path_left, csv_path_right):
+    """Compare 2 CSV files."""
+    df_l = common_df.read_csv(csv_path_left)
```

Comment:
It would be great if there was a quick expectation of what the csv's should contain (in terms of header).  I could see that either being a comment in the docstring or an exception that's thrown with instructions as to how to construct the csv.